agents/mysql/perfschema/perfschema_test.go

```diff
@@ -33,6 +33,11 @@ import (
 	"github.com/percona/pmm-agent/utils/tests"
 )
 
+const (
+	disableQueryExamples = true
+	enableQueryExamples  = false
```

Comment:
There is no need for those constants:
* `newPerfSchema(q, true)` is not clear;
* `newPerfSchema(&newPerfSchemaParams{Querier: q, DisableQueryExamples: true}` is clear;
* `newPerfSchema(&newPerfSchemaParams{Querier: q, DisableQueryExamples: disableQueryExamples}` is less clear because that's too much – it looks like `disableQueryExamples` is a variable that is set to true or false somewhere above :)

Reply:
So we use this variable to pass to setup function and we will have `setup(..., true)` or `setup(..., false)` if we remove this constants.

Reply:
What variable?

Reply:
i meant constants.

Reply:
So how can we avoid passing positional boolean arguments again? ;)

Reply:
new structure?

Reply:
Or _typed_ constant. Or just a different `setup` method.